### ISSUE TITLE
Script to remove adversarial code.

### ISSUE THREAD

**Issue by mfornet** [nearprotocol/nearcore#2027]
All code behind adversarial feature flag should be removed before being merged into master.

**Comment by bowenwang1996** [comment]
Wouldn't this result in a lot of merge conflict when we merge master back into staging?

**Comment by mfornet** [comment]
@SkidanovAlex discussed with @nearmax and @Kouprin and agree that having this tool in our release pipeline will be more detrimental than beneficial. 

It will require a complex pipeline and s pointed out by @bowenwang1996 commits on top of master will be PIA. I understand that you don't want to have any adversarial code on master, now to avoid errors from users running nodes I added an explicit consent request to run a node in adversarial mode. https://github.com/nearprotocol/nearcore/pull/2031/files#diff-d596fd15af4fd5021ba260aeb31cb390R94

**Comment by bowenwang1996** [comment]
For me the main issue is readability. I don't like having adversarial code interspersed with the actual code, but it seems that we cannot find a better solution.

**Comment by ilblackdragon** [comment]
@mfornet @SkidanovAlex where did we end up with adversarial code support?

**Comment by mfornet** [comment]
We decided not to remove adversarial code from codebase, which I strongly support, otherwise it will create painful moments while merging code from one branch to another, and manual intervention will be very hard.

To prevent mistakes nodes compiled with adversary mode enabled will only run [if consent is explicitly set](https://github.com/nearprotocol/nearcore/blob/e34a3dfdbd12d29568a07ccdca8ef343a09f69f0/near/src/main.rs#L100).

Moreover, adversary behavior is not active by default on this binary, but needs to be enabled via RPC calls, so I think it is already very hard to activate adversarial behavior for honest validators by mistake.

Closing this issues.
